tests/unit/app/endpoints/test_feedback.py

```diff
@@ -45,16 +46,19 @@ def test_get_feedback_status(feedback_location):
     assert not feedback.is_feedback_enabled()
 
 
-def test_store_feedback(feedback_location):
+@patch('ols.app.endpoints.feedback.datetime')
```

Comment:
can't we just do:

```suggestion
@patch('ols.app.endpoints.feedback.datetime.utcnow', return_val=datetime(2000,1,1,1,23,45))
```
and avoid the lambda below?

Reply:
`utcnow` returns `callable`, not directly a value

Reply:
https://docs.python.org/3/library/datetime.html#datetime.datetime.utcnow

```
This is like [now()](https://docs.python.org/3/library/datetime.html#datetime.datetime.now), but returns the current UTC date and time, as a naive [datetime](https://docs.python.org/3/library/datetime.html#datetime.datetime) object.
```

??